jbehave-core/src/main/java/org/jbehave/core/model/ExamplesTableFactory.java

```diff
@@ -29,11 +33,16 @@
  */
 public class ExamplesTableFactory {
 
+    public static final Pattern INLINED_PROPERTIES_PATTERN = Pattern.compile("\\{(.*?)\\}\\s*(.*)", DOTALL);
+    public static final ExamplesTable EMPTY = new ExamplesTable("");
+
+    private String propertiesAsString = "";
     private Keywords keywords;
     private final ResourceLoader resourceLoader;
     private final ParameterConverters parameterConverters;
     private final ParameterControls parameterControls;
     private final TableTransformers tableTransformers;
+    private ExamplesTableProperties properties;
```

Comment:
`ExamplesTableFactory` should be immutable, you shouldn't redesign the way of `ExamplesTable` parsing